ci/linux-sgx-gsc-release.jenkinsfile

```diff
@@ -13,6 +13,7 @@ node(node_label) {
         }
     
         dir ("gsc_dev_repo") {
+            currentBuild.description = "distro : " + distro_ver
```

Comment:
Move this before checkout, because we will not get this info if build fails with git clone error

Reply:
Applicable everywhere